tinyagentos/routes/desktop_browser/windows.py

```
@@ -0,0 +1,81 @@
+    active_tab_id: str | None = None
+    state: str  # opaque JSON-serialised tab list / scroll / zoom etc.
+
+
+class WindowsPutBody(BaseModel):
+    windows: list[WindowEntry]
+
+
+@router.get("/api/desktop/browser/windows")
+async def list_browser_windows(
+    request: Request,
+    current_user: dict[str, Any] = Depends(get_current_user),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)